MANIFEST.in

```diff
@@ -1,4 +1,6 @@
-include *requirements.txt
```

Comment:
Was it bad that we were including `dev-requirements.txt` in the package before? 😁

Reply:
Unless we want to run tests from source (but I don't see the point), there is no need for that, no.